storage/client_range_gc_test.go

```diff
@@ -41,6 +41,20 @@ func TestRangeGCQueueDropReplica(t *testing.T) {
 	// Increment the clock's timestamp to expire the leader lease.
 	mtc.manualClock.Increment(int64(storage.DefaultLeaderLeaseDuration) + 1)
 
+	// Make sure the range is not yet removed from the store.
+	numTrials := 10
+	for i := 0; i < numTrials; i++ {
+		store := mtc.stores[1]
+		store.ForceRangeGCScan(t)
+		if _, err := store.GetRange(raftID); err != nil {
+			t.Error("unexpected range removal")
+		}
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Yes, I think we need `Sleep`. Without `Sleep`, the test passed even when `RangeGCQueueUnleasedDuration` is set to 0. I think that's because gorountines for processing ranges didn't run. 

Sure, changed `numTrials` to 3.